node/src/actors/chain_manager/handlers.rs

```diff
@@ -221,9 +221,9 @@ impl Handler<EpochNotification<EveryEpochPayload>> for ChainManager {
             }
         }
 
-        // After block consolidation, commits and reveals that arrive in an incorrect moment
+        // After block consolidation, reveals that arrived in an incorrect moment
         // are processed now
-        for transaction in self.temp_commits_and_reveals.drain(..) {
+        for transaction in self.temp_reveals.drain(..) {
```

Comment:
See below on why this suggestion:
```suggestion
        for reval in self.temp_reveals.drain(..) {
            ctx.notify(AddTransaction {
                transaction: Transaction::Reveal(reveal)
```